src/Encamina.Enmarcha.SemanticKernel/MemoryManager.cs

```diff
@@ -107,7 +130,12 @@ public virtual async IAsyncEnumerable<string> BatchUpsertMemoriesAsync(string co
 
         await foreach (var item in memoryRecordsUniqueIdentifiers)
         {
-            logger.LogInformation($@"Processed memory record {item}.");
+            MemoryStorageEvent?.Invoke(
+                MemoryStorageEventEnum.UpsertBatch.GetEnumDescription(),
```

Comment:
Instead of `MemoryStorageEventEnum.UpsertBatch.GetEnumDescription()` it would be better to use `nameof(MemoryStorageEventEnum.UpsertBatch)`

Reply:
You are right, it is better as you say